clang/test/CXX/special/class.dtor/p7.cpp

```diff
@@ -0,0 +1,69 @@
+// RUN: %clang_cc1 -verify -std=c++26 %s -Wno-defaulted-function-deleted -triple x86_64-linux-gnu
+
+struct NonTrivial {
```

Comment:
WE discussed some new UB aspects that show up with this proposal, can you make sure we have tests that verify we catch those cases during constant evaluation.

Reply:
Where do you want me to add that?

Reply:
class.ctor or class.dtor it is kind of consequence of both, I don't have a strong preference at this point. I just want it part of the test suite.

Reply:
Done.